internal/services/api/deployment_dto.go

```diff
@@ -46,6 +48,8 @@ type fullDeploymentDTO struct {
 	deployment.Deployment
 	ConfigPath string `json:"configurationPath"`
 	SaveName   string `json:"saveName"`
+	LocalID    string `toml:"local_id,omitempty" json:"localId"`
+	AbortedAt  string `toml:"aborted_at,omitempty" json:"abortedAt"`
```

Comment:
These are already available in the `deployment.Deployment` right? Is this not a duplication?

Reply:
Yes, let me track down if this is needed or not. Thanks!

Reply:
Thanks for removing those.